web/profiles/joinup/src/Plugin/Block/RecommendedContentBlock.php

```diff
@@ -94,6 +94,14 @@ public function defaultConfiguration() {
    * {@inheritdoc}
    */
   public function build() {
+    $build = [
+      '#attributes' => ['class' => ['listing', 'listing--grid', 'mdl-grid']],
```

Comment:
I don't think these classes are necessary, because line 127 already adds these.
Also, it seems these classes are not optimal, paragraph goes out of the tiles' alignment:
![attributes_current](https://user-images.githubusercontent.com/1923476/30844384-1c54f006-a28f-11e7-9be7-f71c5fb876fb.png)

You can use the grid "column" classes like this to correct this:
![attributes_paragraph](https://user-images.githubusercontent.com/1923476/30844422-33871722-a28f-11e7-9cd7-31fa5ac859e4.png)